crates/turbopack-core/src/chunk/availability_info.rs

```diff
@@ -1,19 +1,89 @@
+use std::ops::{Add, AddAssign, BitOr, BitOrAssign};
+
 use turbo_tasks::Vc;
 
 use super::available_modules::AvailableAssets;
 use crate::module::Module;
 
+/// This specifies which information is needed from an AvailabilityInfo
+#[turbo_tasks::value(shared)]
+#[derive(Copy, Clone, Debug)]
+pub struct AvailabilityInfoNeeds {
+    pub current_availability_root: bool,
+    pub available_modules: bool,
+}
+
+impl AvailabilityInfoNeeds {
+    pub fn all() -> Self {
+        Self {
+            current_availability_root: true,
+            available_modules: true,
+        }
+    }
+
+    pub fn none() -> Self {
+        Self {
+            current_availability_root: false,
+            available_modules: false,
+        }
+    }
+}
+
+impl BitOr for AvailabilityInfoNeeds {
+    type Output = AvailabilityInfoNeeds;
+
+    fn add(self, rhs: Self) -> Self::Output {
+        match (self, rhs) {
+            (Self::None, rhs) => rhs,
+            (lhs, Self::None) => lhs,
+            (Self::Complete, _) | (_, Self::Complete) => Self::Complete,
+            (Self::Root, Self::Root) => Self::Root,
+            (Self::AvailableModules, Self::AvailableModules) => Self::AvailableModules,
+            (Self::Root, Self::AvailableModules) | (Self::AvailableModules, Self::Root) => {
+                Self::Complete
+            }
+        }
+    }
```

Comment:
Did you forget to delete this?

Reply:
hmm... seems like I also forgot to compile at all...